cogs/generalcog.py

```diff
@@ -65,7 +65,24 @@ async def git(self, ctx, *args):
                     colour=0x1e2327)
         await ctx.send(embed=emb)
 
-
+    @commands.command()
+    async def pokies(self, ctx, *args):
+        # Get all custom Emoji from server
+        allEmotes = self.bot.emojis
+        # Get a random index for each emoji
+        index1, index2, index3 = random.sample(range(0, len(allEmotes)-1), 3)
```

Comment:
`random.sample` will sample without replacement, meaning these `index` are guaranteed to be different (so you will enver get 3 of the same emoji).

Use `random.choice` to take 3 random choices instead. You want something like this:
```py
choose = lambda: random.choice(allEmotes)
a, b, c = choose(), choose(), choose()
```

`choose = lambda: ...` is shorthand for creating a function called "choose" that takes 0 arguments. In the case above each time you invoke `choose()` a random choice is taken from from `allEmotes`

Reply:
No Fun Allowed...
![image](https://user-images.githubusercontent.com/74394004/99060922-a3d0e380-2605-11eb-8ada-5b68cf7b9596.png)

Updated.